### ISSUE TITLE
[Bug/Request] More Power

### ISSUE THREAD

**Issue by Filostorm** [GlobbyPotato/Rockhounding_Chemistry#142]
When power requirement is set above a certain amount (in the config), the Power Station no longer can accept enough power to run the machine.

I would like to be able to increase the power by 10-20x if possible.

**Comment by GlobbyPotato** [comment]
I will give it a look for an improvement. The power consumption is limited because the power station is meant to be a small generator with a limited capacity for a single block unit.

**Comment by Filostorm** [comment]
Thank you! Would adding a higher tier of power station work better then?

**Comment by GlobbyPotato** [comment]
To don't make it too redundant I could just peak the capacity, but if it gets too unrealistic then a new tier could do the job at least for bigger structures. The power station is already a bit too crowded since it deals with fuel, lava, RF, syngas with bidirectional conversions.

**Comment by Filostorm** [comment]
It is pretty crowded. That’s one of the things I have to explain the most to the players of my pack. Simple enough to read once you know how though.

**Comment by gatoborrachon** [comment]
> bidirectional conversions.

@GlobbyPotato  you mean that we can convert Syngas to RF? i thought Syngas was only for Heat energy like lava

**Comment by Primitive-Human** [comment]
> > bidirectional conversions.
> 
> @GlobbyPotato you mean that we can convert Syngas to RF? i thought Syngas was only for Heat energy like lava

Yes, you can use syngas to create heat energy if necessary and RF with a Gas Turbine upgrade. However, this is not necessarily limited to powering RH machines with syngas. As quoted from the guidebook entry on the Power Station:
"The switch in the middle allows to independently provide RF to blocks in front of it. It should be turned off while connected to a structure." 
So you can point the power station to any other RF-consuming block (Thermal Expansion furnace for example), feed the power station syngas and add the gas turbine upgrade and it will produce and provide RF for the TE furnace. Quick testing shows that the power station can produce 2k RF per tick. 
If you "preheat" a power station with coal or lava and then feed it syngas with a gas turbine upgrade,  a gas holder full of syngas can be burned to give 200M RF.
